src/Api/Search.php

```diff
@@ -390,11 +391,20 @@ public static function formatCatalogueOutput($catalogue)
                     $imageUrl = $course['imageUrl'];
                 }
 
+                $summary = $course['description'];
+                if(isset($course['teaserDescription'])){
```

Comment:
This line can be `isset($course['teaserDescription']) && !empty($course['teaserDescription'])` to combine both if conditions

Reply:
done